rosa_cluster_admin/rosa_logging/rosa-accessing-the-service-logs.adoc

```diff
@@ -0,0 +1,17 @@
+:_content-type: ASSEMBLY
+[id="rosa-accessing-the-service-logs"]
+= Accessing the service logs for ROSA clusters
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: rosa-accessing-the-service-logs
+
+toc::[]
+
+[role="_abstract"]
+You can view the service logs for your {product-title} (ROSA) clusters by using {cluster-manager-first} or the {cluster-manager} CLI (`ocm`). The service logs detail cluster events such as the addition of users, groups, and identity providers, load balancer quota updates, and scheduled maintenance upgrades.
```

Comment:
Question, sir: Would it read better with only one "and"?

Reply:
Thank you for highlighting this. I did struggle with this sentence a bit, for that very reason. The problem is, the first three items in the list relate to "the addition of", but the last two do not. If I remove the first "and", then the latter two items read "the addition of load balancer quota updates" and "the addition of scheduled maintenance upgrades", which is not correct. The "the addition of" component only relates to "the addition of users", "the addition of groups" and "the addition of identity providers", so to speak.

I considered resolving this problem by changing this line to a bulleted list, but that might not work for an abstract and they are only a few examples, not a complete list of log message types. I also considered reversing the order of the list, but that does not solve the problem either.

I will have a think about how best to resolve this.

Reply:
@EricPonvelle has provided a great suggestion to resolve this, which I have applied. The sentence is now separated into two, as follows:

> The service logs detail cluster events such as load balancer quota updates and scheduled maintenance upgrades. The logs also show the addition of users, groups, and identity providers.

Thanks Eric!